docs/cli/edgedb_cloud/index.rst

```diff
@@ -28,15 +47,36 @@ log in to EdgeDB Cloud.
     During the Cloud beta, you will only be able to successfully complete
     authentication if you have been invited to the beta.
 
+
+2. Create an instance
+---------------------
+
 Once your login is complete, you may use the other CLI commands to create and
-interact with Cloud instances.
+interact with Cloud instances. To create an Cloud instance, you can use one of
+these commands:
 
-.. list-table::
-    :class: funcoptable
+* :ref:`ref_cli_edgedb_instance_create` with an instance name of
+  ``<github-username>/<instance-name>``.
 
-    * - :ref:`ref_cli_edgedb_cloud_login`
-      - Authenticate to the EdgeDB Cloud and remember the access token locally
-    * - :ref:`ref_cli_edgedb_cloud_logout`
-      - Forget the stored access token
-    * - :ref:`ref_cli_edgedb_cloud_secretkey`
-      - Manage your secret keys
+  .. code-block:: bash
+
+      edgedb instance create <github-username>/<instance-name>
+
+* :ref:`ref_cli_edgedb_project_init` with the ``--server-instance`` option. Set
+  the server instance name to ``<github-username>/<instance-name>``.
```

Comment:
Or, you can specify `<github-username>/<instance-name>` when `project init` prompts you interactively.